UC-Berkeley-Extension/global
# Advising Angular Component

## Asana Ticket(s)
- [Info Sessions](https://app.asana.com/0/1174970560648905/1197472455447401).

## Description
The Advising component for the program pages to go with the leadgen.

## Testing instructions
1. `phing install` should be sufficient if you start from master.
1. Open the Test Article and add a section with the leadgen on the left and the "Angular: Info Sessions Advising" on the right.
1. Confirm that the Get Advising Component shows up
1. Add program ID "1041" and see if the link goes [here](http://go.pardot.com/l/102272/2017-11-03/442h6v)

## Relevant Screenshots/gifs:
![image](https://user-images.githubusercontent.com/17348237/98286623-1fbd9280-1f59-11eb-99fe-d15fa4c64429.png)


## Remaining tasks
- [x] Get image to load from within the module.
